packages/styles/scss/components/data-table/_data-table-skeleton.scss

```diff
@@ -0,0 +1,67 @@
+//
+// Copyright IBM Corp. 2016, 2018
+//
+// This source code is licensed under the Apache-2.0 license found in the
+// LICENSE file in the root directory of this source tree.
+//
+
+@use '../../config' as *;
+@use '../../spacing' as *;
+@use '../../theme' as *;
+@use '../../utilities/convert' as *;
+@use '../../utilities/skeleton' as *;
+
+.#{$prefix}--data-table.#{$prefix}--skeleton th {
```

Comment:
We should make sure to wrap this in some kind of mixin, maybe `data-table-skeleton` or `skeleton`?

Reply:
Fixed 👍